cedarpy/__init__.py

```diff
@@ -158,3 +158,24 @@ def format_policies(policies: str,
     :raises ValueError: if the input policies cannot be parsed
     """
     return _internal.format_policies(policies, line_width, indent_width)
+
+
+def policies_to_json_str(policies: str) -> str:
+    """Convert a cedar policy file to a json cedar policy file.
+
+    :param policies is a str containing the policies to be converted
+
+    :returns the json formatted policy
+    :raises ValueError: if the input policies cannot be parsed
+    """
+    return _internal.policies_to_json_str(policies)
+
+def policies_from_json_str(policies: str) -> str:
+    """Convert a json cedar policy file to a cedar policy file.
+
+    :param policies is a str containing the policies to be converted
+
+    :returns the cedar formatted policy
+    :raises ValueError: if the input policies cannot be parsed
+    """
+    return _internal.policies_from_json_str(policies)
```

Comment:
Thanks for adding these functions.

Do you manage your Cedar policies in JSON?

Reply:
It was to give some sort of parity with the Rust API. In Rust it's possible to parse the policy to the API objects and use this to check what has been generated, e.g. if you want to do some checking on the policy that is not supported by the schema. However returning the Cedar API objects as python objects seemed pretty complex. By allowing cedar policies to be converted to JSON it's probably good enough to be able to see what was generated and do some external validation, the other method to go from json back to cedar native just seemed sensible if you allow from native to json.

Reply:
Thinking about it's probably worth adding a note to the docstring to warn others of this detail